CorsixTH/Lua/filesystem.lua

```diff
@@ -18,7 +18,8 @@ LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
 OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN THE
 SOFTWARE. --]]
 
-local LFS = require "lfs"
+require "lfs"
```

Comment:
Similarly, typically in lua if nothing is done with the require it's not necessary. (It's possible for the require to define globals, but that's considered bad practice.)